### ISSUE TITLE
Switching to the Comments tab in Studios should refresh the comment list

### ISSUE THREAD

**Issue by carljbowman** [LLK/scratch-www#5949]
When navigating between the Projects, Comments, Curators, and Activity tabs the Comment tab does not automatically update when navigating from another tab to Comments. This is inconsistent with the behavior that happens on the Project tab: i.e. if you load a studio page while on the curators tab and a project is added to the studio after initial load, when navigating to the projects tab that new project appears. 

Like the Project tab, the comment tab should be as up-to-date as possible when it's opened and display comments that were added after initial load.

### Steps to Reproduce

1. Open a Studio
2. Go to the Projects Tab
3. Have someone else comment in the Studio you are viewing 
4. After that comment has been posted, navigate to the Comments tab
5. That new comment will not appear unless the page is manually refreshed

**Comment by Accio1** [comment]
This was something that I noticed and was confused by as well, as it didn't match the functionality of the 2.0 studios. Would this also mean that pressing on the comments tab while already on the comments section would refresh the comments as well?

**Comment by carljbowman** [comment]
> Would this also mean that pressing on the comments tab while already on the comments section would refresh the comments as well?

Yes, that should be the case as a side effect of this change.

**Comment by apple502j** [comment]
Isn't this an intentional change to reduce the comment server load? Or is that just a speculation?

**Comment by Accio1** [comment]
> > Would this also mean that pressing on the comments tab while already on the comments section would refresh the comments as well?
> 
> 
> 
> Yes, that should be the case as a side effect of this change.

Makes sense, that was how it worked in 2.0 studios as well, and I prefer it over current functionality.

**Comment by LankyBox01** [comment]
agreed. Also when vieweing a direct link to a comment, we should be able to click the comment tab to refresh & see all the comments
